Formula/dpp.rb

```diff
@@ -0,0 +1,53 @@
+class Dpp < Formula
+  desc "Directly include C headers in D source code"
+  homepage "https://github.com/atilaneves/dpp"
+  url "https://github.com/atilaneves/dpp.git",
+    tag:      "v0.4.9",
+    revision: "55daf91e2c180962de783752b3e4447850b7a151"
+  license "BSL-1.0"
+
+  pour_bottle? only_if: :clt_installed
```

Comment:
maybe add a comment for this line?

Reply:
What should the comment say?

Reply:
Something like what is in `gcc` and `llvm` formulae, mostly just specify the reason why this is needed.

Reply:
Pushed a commit to add both the CLT and Xcode toolchains to rpath, so this shouldn't be needed anymore.